README.md

```diff
@@ -169,10 +169,8 @@ which you need to encode in the YAML file.  For example,
   (E.g. `"EmailAuthBackend,GitHubAuthBackend"`).
 
 **SSL Certificates**.  By default, the image will generate a
-  self-signed cert.  We
-  [will soon also support certbot](https://github.com/galexrt/docker-zulip/issues/120)
-  for this, just like we do in normal Zulip installations
-  (contributions welcome!).
+  self-signed cert.  You can set `SSL_CERTIFICATE_GENERATION: "certbot"` within
+  `docker-compose.yml` to enable automatically-renewed LetsEncrypt certificates.
```

Comment:
We should probably link to the Certbot ToS here, to make sure users know they're accepting them.